docs/ref/contrib/admin/index.txt

```diff
@@ -1576,9 +1579,9 @@ templates used by the :class:`ModelAdmin` views:
 
     .. admonition:: Note
 
-        Any ``choices`` attribute set on the formfield will limited to the form
-        field only. If the corresponding field on the model has choices set,
-        the choices provided to the form must be a valid subset of those
+        Any ``choices`` attributes set on the formfield will be limited to the 
```

Comment:
That's an interesting point, as class Select; can have only once 'choices' selected, but class SelectMultiple can have more than one 'choices' selected. 

Either way, the sentence requires a corrected verb phrase 'will be limited' to handle either cases. This should read:
"Any choices attribute set on the formfield will be limited to the form field only."